Modules/_collectionsmodule.c

```diff
@@ -410,8 +451,19 @@ consume_iterator(PyObject *it)
     return finalize_iterator(it);
 }
 
+/*[clinic input]
+_collections.deque.extend
+
+    deque: dequeobject
+    iterable: object
+    /
+
+Extend the right side of the deque with elements from the iterable
```

Comment:
I think it's not necessary to make the docstring exactly identical. And the current implementation may be missing a period in some docstrings. If we have touched it, I think it's a chance to make it more consistent.

```suggestion
Extend the right side of the deque with elements from the iterable.
```

Reply:
Yes, small amendments like this is fine when applying Argument Clinic.